tools/src/test/java/org/apache/rocketmq/tools/command/acl/UpdateAccessConfigSubCommandTest.java

```diff
@@ -49,7 +49,7 @@ public void testExecute() {
         // Note: Posix parser is capable of handling values that contains '='.
         final CommandLine commandLine =
             ServerUtil.parseCmdLine("mqadmin " + cmd.commandName(), subargs,
-                cmd.buildCommandlineOptions(options), new PosixParser());
+                cmd.buildCommandlineOptions(options), new DefaultParser());
```

Comment:
DefaultParser cannot handle values that contain '='. Need to investigate if there is an option to overcome this issue.

Reply:
See this failure case
1) testExecute(org.apache.rocketmq.tools.command.acl.UpdateAccessConfigSubCommandTest)
java.lang.SecurityException: Test code should never call System.exit()
	at com.google.testing.junit.runner.util.GoogleTestSecurityManager.checkExit(GoogleTestSecurityManager.java:34)
	at java.lang.Runtime.exit(Runtime.java:107)
	at java.lang.System.exit(System.java:973)
	at org.apache.rocketmq.srvutil.ServerUtil.parseCmdLine(ServerUtil.java:56)
	at org.apache.rocketmq.tools.command.acl.UpdateAccessConfigSubCommandTest.testExecute(UpdateAccessConfigSubCommandTest.java:[51](https://github.com/apache/rocketmq/actions/runs/3255627851/jobs/5345142856#step:4:52))

Reply:
> DefaultParser cannot handle values that contain '='. Need to investigate if there is an option to overcome this issue.

I will

Reply:
@lizhanhui DefaultParser can handle values that contain '=', The following validation:
**case1:**
![image](https://user-images.githubusercontent.com/15797831/196077246-5026435e-2700-4960-acc3-76734bc18778.png)
debug result:
![image](https://user-images.githubusercontent.com/15797831/196077419-bd9dab1e-d13f-462f-8624-0d66390deed9.png)
**case 2:**
![image](https://user-images.githubusercontent.com/15797831/196077583-0eb69ad5-ce7a-4d83-a86c-5d61de93e059.png)
debug result:
![image](https://user-images.githubusercontent.com/15797831/196077532-18d80f54-5901-48de-aa4f-281a52ca7093.png)

Reply:
OK. Please look into the failed test case anyway.

Reply:
@lizhanhui  I am watching